docker-compose.yml

```diff
@@ -4,7 +4,12 @@ x-server_settings: &server_settings
   build:
     context: .
     dockerfile: ./dev.Dockerfile
+    args:
+      - OTEL_EXPORTER_OTLP_ENDPOINT=${OTEL_EXPORTER_OTLP_ENDPOINT}
```

Comment:
If you already set the variables in environment variables below, you don't need to pass them as args to the Dockerfile.